core/consensus/babe/has_babe_consensus_digest.hpp

```diff
@@ -3,21 +3,33 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#ifndef KAGOME_CONSENSUS_BABE_HAS_AUTHORITY_SET_CHANGE_HPP
-#define KAGOME_CONSENSUS_BABE_HAS_AUTHORITY_SET_CHANGE_HPP
+#ifndef KAGOME_CONSENSUS_BABE_HAS_BABE_CONSENSUS_DIGEST_HPP
+#define KAGOME_CONSENSUS_BABE_HAS_BABE_CONSENSUS_DIGEST_HPP
 
 #include "primitives/block_header.hpp"
 
+#include "log/logger.hpp"
+
 namespace kagome::consensus::babe {
-  struct HasAuthoritySetChange {
-    HasAuthoritySetChange(const primitives::BlockHeader &block) {
+  struct HasBabeConsensusDigest {
+    static auto logger() {
```

Comment:
Why create a logger every time the function's called, won't storing the logger in a static variable be more appropriate?

Reply:
Because it never (rarely) happens, all blocks are valid